### PR TITLE
Adds Add Funds button to the notification

### PR DESCRIPTION
**Pull request by NejcZdovc**
Needed in https://github.com/brave/brave-core/pull/2512

##### Link / storybook path to visual changes
- 

## Integration
- [ ] Does this contain changes to src/components or src/
  - [ ] Will you publish to npm immediately after this PR, or wait until sometime in the future?
  - [ ] Incompatible API change to something existing _(major version increase)_
  - [ ] Adding new backwards-compatible functionality? _(minor version increase)_
  - [ ] Fixing a bug backwards-compatibly? _(patch version increase)_
  
- [ ] Does this contain changes to src/features for brave-core?
  - [ ] Are there non backwards-compatible changes required for brave-core? **Do not merge until brave-core PR is approvable.** Link to brave-core PR:
  - [ ] Will you create brave-core PR to update to this commit after it is merged?
  - [ ] Wants uplift to brave-core feature branch?
     - When uplift-approved, merge to brave-core-0.VV.x feature branch
     - Create additional brave-core PRs for each feature branch to update commit

### CODE REVIEW

File: src/features/rewards/profile/index.tsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default class Profile extends React.PureComponent<Props, {}> {
         {
           this.getUnverifiedInfo()
         }
-        <StyledVerifiedCheckLink onClick={onRefreshPublisher}>
+        <StyledVerifiedCheckLink onClick={onRefreshPublisher} data-test-id={'unverified-check-button'}>
```
Comment (jasonrsadler) [inline src/features/rewards/profile/index.tsx]: Are you going to need this across the other instances or just the "clickable"

File: src/features/rewards/walletWrapper/index.tsx [hunk 2]
```diff
@@ -83,7 +83,17 @@ export interface ActionWallet {
...
+  'ads' |
+  'ads-launch' |
+  'backupWallet' |
+  'contribute' |
+  'grant' |
+  'insufficientFunds' |
+  'tipsProcessed' |
+  'error' |
+  'pendingContribution' |
+  ''
```
Comment (jasonrsadler) [inline src/features/rewards/walletWrapper/index.tsx]: I know this is just alignment mod but do we need the `''`. The `default` case already handles

File: src/features/rewards/profile/index.tsx (hunk 1 above)
Comment (NejcZdovc) [reply src/features/rewards/profile/index.tsx]: it's just so that I can trigger a click

File: src/features/rewards/walletWrapper/index.tsx (hunk 2 above)
Comment (NejcZdovc) [reply src/features/rewards/walletWrapper/index.tsx]: isn't default case right for this use case, when you have empty string as type can't be null

**Pull request merged**